src/Debug/skeletonViewer.ts

```diff
@@ -129,7 +387,8 @@ export class SkeletonViewer {
 
         //Defaults
         options.pauseAnimations = options.pauseAnimations ?? true;
-        options.returnToRest = options.returnToRest ?? true;
+        options.currentStateIsRestPose = options.currentStateIsRestPose ?? false;
+        options.returnToRest = (options.currentStateIsRestPose) ? false : options.returnToRest ?? (options.currentStateIsRestPose) ? false : true;

```

Comment:
Maybe `options.returnToRest = options.currentStateIsRestPose ? false : options.returnToRest ?? !options.currentStateIsRestPose;` is easier to read?